<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>1068</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>华文新魏</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>郭子毓计算机图形学设计课项目——目标检测与车牌识别</string>
  </property>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>1321</width>
     <height>701</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文新魏</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>图像处理</string>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>231</width>
      <height>291</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文新魏</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>车牌识别</string>
    </property>
    <widget class="QPushButton" name="btn_plc">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文新魏</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>选择文件</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_pls">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文新魏</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>SSD强化识别</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_plr">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文新魏</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Retinanet强化识别</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ply">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文新魏</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>YoloV3强化识别</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>1021</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文新魏</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>目标检测模型效果展示</string>
    </property>
    <widget class="QGraphicsView" name="po1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>482</width>
       <height>272</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文新魏</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Origin</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="po2">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>50</y>
       <width>482</width>
       <height>272</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文新魏</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Result</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>231</width>
      <height>291</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文新魏</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>目标检测</string>
    </property>
    <widget class="QPushButton" name="btn_pos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文新魏</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>SSD检测</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_por">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文新魏</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Retinanet检测</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_poy">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文新魏</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>YoloV3检测</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_poc">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文新魏</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>选择文件</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>1021</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文新魏</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>车牌识别模型效果展示</string>
    </property>
    <widget class="QGraphicsView" name="pl1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>482</width>
       <height>272</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文新魏</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Origin</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="pl2">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>50</y>
       <width>482</width>
       <height>272</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文新魏</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Result</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>710</y>
     <width>601</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>视频处理</string>
   </property>
   <widget class="QGroupBox" name="groupBox_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>271</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="title">
     <string>目标检测</string>
    </property>
    <widget class="QPushButton" name="btn_voc">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>选择文件</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_vos">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>SSD检测</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_vor">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Retinanet检测</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_voy">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>YoloV3检测</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_10">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>271</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="title">
     <string>车牌识别</string>
    </property>
    <widget class="QPushButton" name="btn_vlc">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>选择文件</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_vls">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>SSD检测</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_vlr">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Retinanet检测</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_vly">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>YoloV3检测</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_11">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>720</y>
     <width>661</width>
     <height>321</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>程序运行情况提示栏</string>
   </property>
   <widget class="QTextBrowser" name="textshow">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>521</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'华文新魏'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_poc</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>poc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>11</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_pos</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>pos()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>11</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_por</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>por()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>12</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_poy</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>poy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>9</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_plc</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>plc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>464</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_pls</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>pls()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>535</y>
    </hint>
    <hint type="destinationlabel">
     <x>10</x>
     <y>523</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_plr</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>plr()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>583</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>577</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_ply</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>ply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>647</y>
    </hint>
    <hint type="destinationlabel">
     <x>11</x>
     <y>624</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_voc</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>voc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>801</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>783</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_vos</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>vos()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>866</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>855</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_vor</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>vor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>940</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>927</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_voy</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>voy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>985</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>1047</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_vlc</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>vlc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>496</x>
     <y>799</y>
    </hint>
    <hint type="destinationlabel">
     <x>650</x>
     <y>773</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_vls</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>vls()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>865</y>
    </hint>
    <hint type="destinationlabel">
     <x>659</x>
     <y>849</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_vlr</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>vlr()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>505</x>
     <y>928</y>
    </hint>
    <hint type="destinationlabel">
     <x>651</x>
     <y>900</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_vly</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>vly()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>497</x>
     <y>992</y>
    </hint>
    <hint type="destinationlabel">
     <x>655</x>
     <y>983</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>poc()</slot>
  <slot>pos()</slot>
  <slot>por()</slot>
  <slot>poy()</slot>
  <slot>plc()</slot>
  <slot>pls()</slot>
  <slot>plr()</slot>
  <slot>ply()</slot>
  <slot>voc()</slot>
  <slot>vos()</slot>
  <slot>vor()</slot>
  <slot>voy()</slot>
  <slot>vlc()</slot>
  <slot>vls()</slot>
  <slot>vlr()</slot>
  <slot>vly()</slot>
 </slots>
</ui>
